docs/user/ToolLimitations.md

```diff
@@ -0,0 +1,44 @@
+# Capabilities/Limitations
+
+## Global Considerations
+
+-   OpenROAD v1.0 production will be focused on the tapeout mentioned in
+    the above introduction. Features will be implemented in priority
+    order based on our sponsor requirement to make the chosen design
+    manufacturable. In Phase 2 of the IDEA program, the OpenROAD tool feature
+    set will be rounded out and more of the project's flow and tool research
+    objectives will be addressed.
+
+-   Each new design entablement (foundry process/PDK, library, IPs) will
+    require setup via configuration files, one-time characterizations,
+    etc. as documented with the tool. Examples include (i) the setup of PDN
+    generation, (ii) the creation of "wrapped LEF abstracts" for cells and/or
+    macros to comply with Generic Node Entablement (see Routing, below), and
+    (iii) the creation of characterized lookup tables to guide CTS buffering.
+
```

Comment:
Very outdated- can we update atleast version no? Also since we are in Phase 2b- we should make correct statements for tool status on PPA etc. Next pass..

Reply:
Will remove since this was kept only for history.